get_golang_versions.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# -*- mode: bash; -*-
+
+# File: get_golang.sh
+# Copyright (C) 2019 Gravitational Inc.
+# Description: Find 3 latest major Golang releases
```

Comment:
Nit: I think this repo is open, so should probably have our regular license header??